### ISSUE TITLE
[Enhancement] Abillity to Scale RTG Modes Pixel Perfect

### ISSUE THREAD

**Issue by nickkeane** [tonioni/WinUAE#218]
I have "RTG Board -> Always center" checked on the UAE card, which produces a pixel perfect exact resolution (UAE 640x512), but it's small on screen.

Using native modes, the Filter settings (i.e. Filter -> Horiz. size, Vert. size) allow scaling up 2x to be done no problem (or with Auto Integer scaling), but the Filter settings for RTG mode are always greyed out and not changeable, whether filter drop down is No scaling, Default, or Automatic center (is this a bug or simply unsupported?). Without Always Center checked, RTG modes scale to fit the full height of my display (1920x1080), which produces significant pixel distortion.

Basically looking to produce a 1280x1024 centered display in RTG mode, upscaled 2x from 640x512, but i'm not sure what settings to change in order to do that or if it's possible.

**Comment by tonioni** [comment]
RTG integer multiplier support will be implemented in next version.

**Comment by tonioni** [comment]
Implemented in 4.10
